lib/krikri/enrichments/timespan_split.rb

```diff
@@ -61,7 +61,10 @@ def parse_labels(labels)
 
     def span_from_date(date)
       return [nil, nil] if date.nil?
-      return [date, date] if date.is_a? Date
+      if date.is_a?(Date)
+        return [date, date] if date.precision == :day
+        return [date, (date.succ - 1)]
```

Comment:
`#succ` is overridden by `EDTF` to allow for month/year precision.  So this is either:
  - +1 day, -1 day
  - +1 month, -1 day
  - +1 year, -1 day

For example: If the date is `1999` with year precision, this sets the range to Jan 1, 1999 - Dec. 31 1999.